src/tls.cpp

```diff
@@ -121,6 +121,7 @@ int tls_init(void) {
         goto error_free_ctx;
     }
 
+    SSL_CTX_set_options(ctx, SSL_OP_ENABLE_KTLS);
```

Comment:
Could you please wrap this call with openssl version check? Starting from 3.0.0 stable we need to call this and ignore for older versions. In this case we can support openssl-1.x.x version as well. Sockperf doesn't have to provide KTLS functionality and in most distributions you won't find openssl-3.0.0 out of box right now.